app/src/main/java/nl/tudelft/bbw/controller/BlockController.java

```diff
@@ -261,4 +278,35 @@ private boolean verifySignature(byte[] signature, byte[] message) {
         }
     }
 
+    //For debugging purpose
+    public Database getDatabase()
```

Comment:
You are absolutely right. The method has been moved and the getDatabase method has been deleted.